src/clj/pyregence/match_drop.clj

```diff
@@ -511,7 +523,7 @@
                         :message             (str "Match Drop #" match-job-id " initiated from Pyrecast.")
                         :elmfire-done?       false
                         :gridfire-done?      false
-                        :dps-request         {}
+                        :dps-request         match-drop-inputs
```

Comment:
This fixes `Lat, Lon` in the dashboard, together with the `common-args           (or common-args dps-request)` fix in the dashboard namespace.
This is supposed to be temporary until we clean it all up: use only kubernetes and not runway anymore: so then we can remove some columns from the database (from the `match_jobs` table), store only the one http request we do call etc.